Source/Moq.Analyzers/AnalyzerReleases.Unshipped.md

```diff
@@ -0,0 +1,17 @@
+﻿; Unshipped analyzer release
+; https://github.com/dotnet/roslyn-analyzers/blob/main/src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md
+
+### Changed Rules
+
+Documentation links added for every rule.
+
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+Moq1000 | Moq | Warning | NoSealedClassMocksAnalyzer, [Documentation](https://github.com/rjmurillo/moq.analyzers/blob/main/docs/rules/Moq1000.md)
```

Comment:
As of 0.0.9 these are all shipped, no?

Reply:
Yeah, changes incoming here.

The idea was that we added the HelpUri, which made it a "changed rule", but the analyzer only cares about (Id, Category, Severity).

Over the releases we've changed which analyzer used which ID, so I'll need to adjust.

Reply:
> Over the releases we've changed which analyzer used which ID, so I'll need to adjust.

🤦

Reply:
OK, the files are now correct according to the analyzer. However, we have a history issue: Over time we've remapped IDs to analyzers, so adding the documentation links into the release tracking document means either:

1. Adding a link to the _current_ ID, which may not be what the analyzer checked at the time
2. Adding a link to the _original_ ID, which will mean the ID in the document won't match the help link

Thoughts?